dev/archery/archery/crossbow/cli.py

```diff
@@ -155,6 +155,49 @@ def submit(obj, tasks, groups, params, job_prefix, config_path, arrow_version,
         click.echo('Pushed job identifier is: `{}`'.format(job.branch))
 
 
+@crossbow.command()
+@click.option('--base-branch', '-b', default="master",
+              help='Set base branch for the PR.')
+@click.option('--arrow-remote', '-r', default=None,
```

Comment:
How about `--remote` instead of `--arrow-remote`?
It seems that `arrow-` is redundant in this context.